packages/tracing-internal/src/browser/metrics/index.ts

```diff
@@ -245,7 +245,7 @@ export function addPerformanceEntries(transaction: Transaction): void {
     if (typeof responseStartTimestamp === 'number' && transactionStartTime) {
       DEBUG_BUILD && logger.log('[Measurements] Adding TTFB');
       _measurements['ttfb'] = {
-        value: (responseStartTimestamp - transactionStartTime) * 1000,
+        value: Math.max(responseStartTimestamp - transactionStartTime, 0) * 1000,
```

Comment:
Can we add a comment to here? As well as a test?

Reply:
Yup good call - done with 195a3099caa1b7fbaf2e9ca3a037b40de79d43a7